site/src/content/docs/migration.mdx

```diff
@@ -1,23 +1,27 @@
 ---
-layout: docs
 title: Migrating to v5
 description: Track and review changes to the Bootstrap source files, documentation, and components to help you migrate from v4 to v5.
-group: migration
 aliases: "/migration/"
 toc: true
 ---
 
+## v5.3.6
```

Comment:
I've added it like it was done from Jekyll to Hugo back then, but it can't be removed if not necessary.